docs/man/containerd.1.md

```diff
@@ -1,5 +1,9 @@
 # containerd 1 01/29/2018
 
+## NAME
+
+containerd - high performance container runtime
```

Comment:
`containerd - An industry-standard container runtime with an emphasis on simplicity, robustness and portability`? (from https://containerd.io/)

Reply:
@AkihiroSuda  I copied the sentence from `containerd -h` but I'm fine to say whatever.

Reply:
> `containerd - An industry-standard container runtime with an emphasis on simplicity, robustness and portability`? (from https://containerd.io/)

+1